app/core/store/modules/seasonalLeague.js

```diff
@@ -163,7 +169,10 @@ export default {
         }
         return []
       }
-    }
+    },
+    codePointsPlayerCount(state) {
+      return state.codePointsRankingsForLeague.playerCount;
```

Comment:
Not sure if it's here or in the template, but we should use `playerCount.toLocaleString()` to add the appropriate decimal separators in there. Screenshots just 222265 players, but for readability when numbers get past 10K, we want that comma: 222,265 players.

This should apply to other places we display counts, it's just that there are rarely such large numbers! 🥳

Reply:
made the change, merging.